### ISSUE TITLE
install on 2.0.6 yii2 fails

### ISSUE THREAD

**Issue by iBotPeaches** [yii2tech/ar-file#1]
```
  Problem 1
    - Installation request for yii2tech/ar-file ^1.0 -> satisfiable by yii2tech/ar-file[1.0.0].
    - yii2tech/ar-file 1.0.0 requires yii2tech/file-storage 1.0.x-dev -> no matching package found.
```

(that was on `stable`)

Switching to `dev` minimum-stability

```
  Problem 1
    - Installation request for yii2tech/file-storage == 1.0.0.0 -> satisfiable by yii2tech/file-storage[1.0.0].
    - yii2tech/ar-file 1.0.0 requires yii2tech/file-storage 1.0.x-dev -> satisfiable by yii2tech/file-storage[1.0.x-dev].
    - yii2tech/ar-file 1.0.x-dev requires yii2tech/file-storage 1.0.x-dev -> satisfiable by yii2tech/file-storage[1.0.x-dev].
    - Conclusion: don't install yii2tech/file-storage 1.0.x-dev
    - Installation request for yii2tech/ar-file ^1.0 -> satisfiable by yii2tech/ar-file[1.0.0, 1.0.x-dev].
```

Not sure how to fix.

**Comment by klimov-paul** [comment]
Problem caused by;
https://github.com/yii2tech/ar-file/blob/master/composer.json#L21

As a walk around you can specify following options at your `composer.json`:

``` json
"minimum-stability": "dev",
"prefer-stable": true,
```

**Comment by klimov-paul** [comment]
Resolved by commit 07c5a6abe10e26bd59fa40ca87b9aef63d692ce3
